README.md

```diff
@@ -315,19 +315,6 @@ dema
     --overbought_rsi=<value>  sold when RSI exceeds this value (default: 80)
     --noise_level_pct=<value>  do not trade when short ema is with this % of last short ema (default: 2)
 
-dema
```

Comment:
I'd rename this to `dema-lessnoise` and update https://github.com/DeviaVir/zenbot/blob/unstable/extensions/strategies/dema-lessnoise/strategy.js#L6 instead of removing this, otherwise it'll be back in a next iteration